### PR TITLE
Extension setup

### PR DESCRIPTION
**Pull request by beamb**
Aside from having updated the README file, the code has been setup for our React app to run as a Google Chrome extension. Setup instructions are also available in the README file. The app itself has not yet undergone changes and still displays its default state (with the React logo and instructions).

### CODE REVIEW

File: public/manifest.json [hunk 1]
```diff
@@ -1,25 +1,25 @@
...
-      "src": "logo192.png",
-      "type": "image/png",
-      "sizes": "192x192"
-    },
-    {
-      "src": "logo512.png",
-      "type": "image/png",
-      "sizes": "512x512"
-    }
+  "name": "Microlearning",
```
Comment (mircealungu) [inline public/manifest.json:2]: too generic?

File: public/manifest.json [hunk 2]
```diff
@@ -1,25 +1,25 @@
...
+  },
+  "browser_action": {
+    "default_icon": "logo512.png",
+    "default_popup": "index.html",
+    "default_title": "Microlearning – programming practice"
+  },
+  "permissions": [
+    "activeTab",
+    "alarms",
+    "clipboardRead",
```
Comment (mircealungu) [inline public/manifest.json:18]: do we really need all these permissions?

**Review by madsnedergaard — COMMENTED** [review]
Not much to comment on here, but it works ¯\\\_(ツ)\_/¯

If you don't have it already, I'd suggest that you add a branch protection rule like this, so you don't "accidentally" commit things to the wrong branch:

Screenshot 2020-10-21 at 13 05 11

image

**Review by Akons97 — APPROVED** [review]
As Mads said not much to comment on here but so far so good :)

**Comment by madsnedergaard** [comment]
Next time, it would be great if you can address any comments in the PR and merge it, before continuing work on `develop` :)

File: public/manifest.json (hunk 1 above)
Comment (beamb) [reply public/manifest.json:2]: We'll definitely come up with a snappier name for our app once it has started to take more shape :-)

File: public/manifest.json (hunk 2 above)
Comment (beamb) [reply public/manifest.json:18]: Not yet, but we will so they're also there as a reminder of the different Google APIs we need/want to implement for our app to work as per the requirements.

**Pull request merged**